### DISCUSSION TITLE
Anti-kick

### DISCUSSION THREAD

**Discussion by Gloktober** [winstxnhdw/lc-hax#205] [General]
would it be possible to make it so anti-kick doesn't stop working everytime someone leaves or join ?

**Comment by zx3800** [comment]
for me actually, it breaks when someone DOESNT leave or join.
well -- it doesnt show the playernames and people only show up as Player #1-3 regardless of the number of players

**Comment by winstxnhdw** [comment]
i'm working on it

**Comment by winstxnhdw** [comment]
Hey all, I've improved anti-kick in the latest version. You should be completely unkickable now. Let me know how it goes!

**Reply by Gloktober** [reply]
works very good ! haven't had any issues when people leave or join the lobby. Great work

**Comment by winstxnhdw** [comment]
For future readers:

No, anti-kick still has its previous caveats.
